metadata/batch_bigquery_template_metadata.json

```diff
@@ -59,6 +59,12 @@
       "helpText": "The string name of the column that contains a datetime. The column should be of type TIMESTAMP in the SQL schema.",
       "label": "Date column"
     },
+    {
+      "name": "segment_column",
```

Comment:
this should be plural with a validation regex for a json list, or a fancier one that is a json list or a string if you want to allow both formats. I don't feel strongly there.

JSON list or csv, whatever is consistent. If there is already json in the template somewhere then I'd say keep it json.